City-of-Helsinki/hel-fi-drupal-grants
# fix: AU-895: form styles for kuva projekti

# AU-895


## What was done


* Added a lot of fieldsets for the form.
* Added two new fields to the form - the henkilötyövuodet on page 3.
* Added some help-texts and other texts

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-895-kuva-projektihakemus-webform-styles`
  * `make fresh`
* Run `make drush-cr`

## How to Test

* Go to form. Navigate through form. See that form matches documents mentioned in the Task:
* [ ] The Excel
* [ ] The Abstract
